sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/EventHubsException.cs

```diff
@@ -10,6 +10,22 @@ namespace Microsoft.Azure.EventHubs
     /// </summary>
     public class EventHubsException : Exception
     {
+        /// <summary>
+        ///Enumerates the types of error level for the messaging communication.
+        /// </summary>
+        public enum ErrorLevelType
```

Comment:
nit:  Semantically, `level` typically tends to relate to a degree of severity.  This is intended to offer context around the source (cause?) of the error.  Not sure how strongly you feel about aligning to the semantics used by Service Bus, but a name similar to `ErrorSource` may be more meaningful.

Reply:
Agreed, I can see that when you pointed it out :)

---
In reply to: [311763970](https://github.com/Azure/azure-sdk-for-net/pull/7194#discussion_r311763970) [](ancestors = 311763970)